lineman/app/models/base_endpoints_interface.coffee

```diff
@@ -0,0 +1,22 @@
+angular.module('loomioApp').factory 'BaseEndpointsInterface', (RestfulClient) ->
```

Comment:
This fulfills a need that came up a couple times in the past, which is that we want to be able to hit an endpoint without having to actually try to store anything to the record store. This means I don't have to make `invitable_model.coffee` or `invitation_model.coffee`, which would just be cruft.